### PR TITLE
Rn v5 integrations

### PR DESCRIPTION
**Pull request by ivosam3**
Updated all features for React Native SDK v5

### CODE REVIEW

File: src/content/docs/sdk/react-native/features/att.mdx [hunk 1]
```diff
@@ -2,8 +2,162 @@
...
+Adjust.getAppTrackingAuthorizationStatus();
+```
+
+### [Example](example-1)
+
+This example shows how to collect the user's authorization status and convert it to a `string`. This information is assigned to a variable called `authorizationStatus` and passed as a session partner parameter with the key `"status"`.
+
+```jsx
+var authorizationStatus = String(Adjust.getAppTrackingAuthorizationStatus());
+Adjust.addSessionPartnerParameter("status", authorizationStatus);
```
Comment (Aditi3) [inline src/content/docs/sdk/react-native/features/att.mdx:125]:
```suggestion
Adjust.getAppTrackingAuthorizationStatus(function(status) {
      Adjust.addSessionPartnerParameter("status", status)
});
```

File: src/content/docs/sdk/react-native/features/callbacks.mdx [hunk 2]
```diff
@@ -50,7 +52,266 @@ const adjustConfig = new AdjustConfig(
    "{YourAppToken}",
    AdjustConfig.EnvironmentSandbox,
 );
-//...
+
+adjustConfig.setSessionTrackingSucceededCallbackListener(
```
Comment (Aditi3) [inline src/content/docs/sdk/react-native/features/callbacks.mdx:56]: Callback renaming - 

- `setSessionTrackingSucceededCallbackListener` to `setSessionTrackingSucceededCallback`

- `setSessionTrackingFailedCallbackListener` to `setSessionTrackingFailedCallback`

- `setEventTrackingFailedCallbackListener` to `setEventTrackingFailedCallback`

- `setEventTrackingFailedCallbackListener` to `setEventTrackingFailedCallback`

File: src/content/docs/sdk/react-native/features/purchase-verification.mdx [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+If you use revenue events to measure your purchases in Adjust, you should use the `AdjustEvent` class. If you only want to verify a purchase but don't want to associate it with an event, use the `AdjustAppStorePurchase` or `AdjustPlayStorePurchase` class.
+
+</Callout>
+
+When you send purchase information with the Adjust SDK, Adjust does the following:
+
+1. Sends the information to the relevant store and waits for a status response.
+2. Forwards the status response to the Adjust SDK.
+
+You can access the purchase verification status by using a callback. Results are returned as `AdjustPurchaseVerificationInfo` objects containing the following properties:
```
Comment (Aditi3) [inline src/content/docs/sdk/react-native/features/purchase-verification.mdx:26]:
```suggestion
You can access the purchase verification status by using a callback. Results are returned as `AdjustPurchaseVerificationResult` objects containing the following properties:
```

File: src/content/docs/sdk/react-native/features/purchase-verification.mdx [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+
+## [Verify purchase and record event](verify-purchase-and-record-event)
+
+<Tabs>
+<Tab title="App Store" icon="StoreAppStore" sync="appstore">
+
+<CodeBlock title="Method signature">
+
+```js
+verifyAndTrackAppStorePurchase: (adjustEvent: AdjustEvent, callback: (verificationInfo: AdjustPurchaseVerificationInfo) => void) => void
```
Comment (Aditi3) [inline src/content/docs/sdk/react-native/features/purchase-verification.mdx:40]:
```suggestion
verifyAndTrackAppStorePurchase: (adjustEvent: AdjustEvent, callback: (verificationResult: AdjustPurchaseVerificationResult) => void) => void
```

File: src/content/docs/sdk/react-native/features/purchase-verification.mdx [hunk 5]
```diff
@@ -0,0 +1,221 @@
...
+</CodeBlock>
+
+To send a revenue event for verification and listen for the purchase verification status, follow these steps:
+
+1. Instantiate an `AdjustEvent` object with the your event token and set the following parameters:
+   -  `ProductId` (`string`): The product identifier of the item that was successfully purchased.
+   -  `TransactionId` (`string`): The ID of the transaction you want to verify.
+2. Call the `Adjust.verifyAndTrackPlayStorePurchase` method with the following arguments:
+   -  `adjustEvent` (`AdjustEvent`): Your instantiated event object.
+   -  `callback` (`function`): A delegate callback function that receives an `AdjustPurchaseVerificationInfo` object as an argument.
```
Comment (Aditi3) [inline src/content/docs/sdk/react-native/features/purchase-verification.mdx:52]:
```suggestion
   -  `callback` (`function`): A delegate callback function that receives an `AdjustPurchaseVerificationResult` object as an argument.
```

File: src/content/docs/sdk/react-native/features/purchase-verification.mdx [hunk 6]
```diff
@@ -0,0 +1,221 @@
...
+});
+```
+
+</Tab>
+<Tab title="Google Play" icon="PlatformGooglePlay" sync="playstore">
+
+<CodeBlock title="Method signature">
+
+```js
+verifyAndTrackPlayStorePurchase: (adjustEvent: AdjustEvent, callback: (verificationInfo: AdjustPurchaseVerificationInfo) => void) => void
```
Comment (Aditi3) [inline src/content/docs/sdk/react-native/features/purchase-verification.mdx:74]:
```suggestion
verifyAndTrackPlayStorePurchase: (adjustEvent: AdjustEvent, callback: (verificationResult: AdjustPurchaseVerificationResult) => void) => void
```

File: src/content/docs/sdk/react-native/features/purchase-verification.mdx [hunk 7]
```diff
@@ -0,0 +1,221 @@
...
+</CodeBlock>
+
+To send a revenue event for verification and listen for the purchase verification status, follow these steps:
+
+1. Instantiate an `AdjustEvent` object with the your event token and set the following parameters:
+   -  `productId` (`string`): The ID of the product that has been purchased.
+   -  `purchaseToken` (`string`): The purchase token associated with the purchase.
+2. Call the `Adjust.verifyAndTrackPlayStorePurchase` method with the following arguments:
+   -  `adjustEvent` (`AdjustEvent`): Your instantiated event object.
+   -  `callback` (`function`): A delegate callback function that receives an `AdjustPurchaseVerificationInfo` object as an argument.
```
Comment (Aditi3) [inline src/content/docs/sdk/react-native/features/purchase-verification.mdx:86]:
```suggestion
   -  `callback` (`function`): A delegate callback function that receives an `AdjustPurchaseVerificationResult` object as an argument.
```

File: src/content/docs/sdk/react-native/features/purchase-verification.mdx [hunk 8]
```diff
@@ -0,0 +1,221 @@
...
+</CodeBlock>
+
+To send a standalone Play Store purchase and listen for the purchase verification status, follow these steps:
+
+1. Instantiate an `AdjustPlayStorePurchase` with the following arguments:
+   -  `productId` (`string`): The ID of the product that has been purchased.
+   -  `purchaseToken` (`string`): The purchase token associated with the purchase.
+2. Call the `Adjust.verifyPlayStorePurchase` method with the following arguments:
+   -  `purchase` (`AdjustPlayStorePurchase`): Your instantiated purchase object.
+   -  `verificationResultCallback` (`function`): A delegate callback function that receives an `AdjustPurchaseVerificationInfo` object as an argument.
```
Comment (Aditi3) [inline src/content/docs/sdk/react-native/features/purchase-verification.mdx:156]:
```suggestion
   -  `verificationResultCallback` (`function`): A delegate callback function that receives an `AdjustPurchaseVerificationResult` object as an argument.
```

File: src/content/docs/sdk/react-native/features/purchase-verification.mdx [hunk 9]
```diff
@@ -0,0 +1,221 @@
...
+
+## [Only verify purchase](only-verify-purchase)
+
+<Tabs>
+<Tab title="App Store" icon="StoreAppStore" sync="appstore">
+
+<CodeBlock title="Method signature">
+
+```js
+verifyAppStorePurchase: (purchase: AdjustAppStorePurchase, callback: (verificationInfo: AdjustPurchaseVerificationInfo) => void) => void
```
Comment (Aditi3) [inline src/content/docs/sdk/react-native/features/purchase-verification.mdx:113]:
```suggestion
verifyAppStorePurchase: (purchase: AdjustAppStorePurchase, callback: (verificationResult: AdjustPurchaseVerificationResult) => void) => void
```

File: src/content/docs/sdk/react-native/features/purchase-verification.mdx [hunk 10]
```diff
@@ -0,0 +1,221 @@
...
+</CodeBlock>
+
+To send a standalone App Store purchase and listen for the purchase verification status, follow these steps:
+
+1. Instantiate an `AdjustAppStorePurchase` object with the following arguments:
+   -  `productId` (`string`): The product identifier of the item that was successfully purchased.
+   -  `transactionId` (`string`): The ID of the transaction you want to verify.
+2. Call the `Adjust.verifyAppStorePurchase` method with the following arguments:
+   -  `purchase` (`AdjustAppStorePurchase`): Your instantiated event object.
+   -  `callback` (`function`): A delegate callback function that receives an `AdjustPurchaseVerificationInfo` object as an argument.
```
Comment (Aditi3) [inline src/content/docs/sdk/react-native/features/purchase-verification.mdx:125]:
```suggestion
   -  `callback` (`function`): A delegate callback function that receives an `AdjustPurchaseVerificationResult` object as an argument.
```

File: src/content/docs/sdk/react-native/features/purchase-verification.mdx [hunk 11]
```diff
@@ -0,0 +1,221 @@
...
+});
+```
+
+</Tab>
+<Tab title="Google Play" icon="PlatformGooglePlay" sync="playstore">
+
+<CodeBlock title="Method signature">
+
+```js
+verifyPlayStorePurchase: (purchase: AdjustPlayStorePurchase, callback: (verificationInfo: AdjustPurchaseVerificationInfo) => void) => void
```
Comment (Aditi3) [inline src/content/docs/sdk/react-native/features/purchase-verification.mdx:144]:
```suggestion
verifyPlayStorePurchase: (purchase: AdjustPlayStorePurchase, callback: (verificationResult: AdjustPurchaseVerificationResult) => void) => void
```

File: src/content/docs/sdk/react-native/features/skad.mdx [hunk 12]
```diff
@@ -2,8 +2,221 @@
...
+      console.log("Conversion value: " + cconversionData);
+   });
+
+   Adjust.initSdk(adjustConfig);
+};
+
+export default App;
+```
+
+## [SKAdNetwork 4.0 callbacks](skadnetwork-4-0-callbacks)
```
Comment (Aditi3) [inline src/content/docs/sdk/react-native/features/skad.mdx:141]: SKAdNetwork 4 documentation is not needed under the integration guide, I guess.

File: src/content/docs/sdk/react-native/features/ad-revenue.mdx [hunk 13]
```diff
@@ -2,8 +2,306 @@
...
+-  _`"network1"`_ as the ad revenue network
+-  _`"unit1"`_ as the ad revenue unit
+-  _`"banner"`_ as the ad revenue placement
+-  A callback parameter: `"key1" = "value1"`
+-  A partner parameter: `"key2" = "value2"`
+
+```jsx
+var adjustAdRevenue = new AdjustAdRevenue(
+   AdjustConfig.AdRevenueSourceAppLovinMAX,
+);
```
Comment (Aditi3) [inline src/content/docs/sdk/react-native/features/ad-revenue.mdx:290]:
```suggestion
var adjustAdRevenue = new AdjustAdRevenue("applovin_max_sdk");
```

File: src/content/docs/sdk/react-native/index.mdx [hunk 14]
```diff
@@ -21,11 +24,305 @@ To use the Adjust SDK in your React Native app, you need to add it to your proje
 1. Download the React Native library using one of the following options.
 
 <Tabs>
+<Tab title="npm">
+
+Run the following command on your terminal:
+
+```console
+$ npm install react-native-adjust --save
```
Comment (Aditi3) [inline src/content/docs/sdk/react-native/index.mdx:32]:
```suggestion
$ npm install react-native-adjust@5.0.0 --save
```

Comment (Sporiff) [reply src/content/docs/sdk/react-native/index.mdx]: @Aditi3 I've updated this with a template that will pull the latest version from GitHub at build.

File: src/content/docs/sdk/react-native/features/ad-revenue.mdx [hunk 15]
```diff
@@ -2,8 +2,306 @@
...
+
+-  `source` (`string`): The source of the ad revenue. See the table below for available sources
+
+<Table>
+
+| Parameter               | Source            |
+| ----------------------- | ----------------- |
+| `applovin_max_sdk`      | AppLovin MAX      |
+| `admob_sdk`             | AdMob             |
+| `ironsource_sd`         | ironSource        |
```
Comment (uerceg) [inline src/content/docs/sdk/react-native/features/ad-revenue.mdx:40]:
```suggestion
| `"ironsource_sdk"`         | ironSource        |
```

File: src/content/docs/sdk/react-native/features/ad-revenue.mdx [hunk 16]
```diff
@@ -2,8 +2,306 @@
...
+| `admost_sdk`            | AdMost            |
+| `unity_sdk`             | Unity             |
```
Comment (uerceg) [inline src/content/docs/sdk/react-native/features/ad-revenue.mdx:42]:
```suggestion
| `"unity_sdk"`             | Unity             |
```

File: src/content/docs/sdk/react-native/features/ad-revenue.mdx [hunk 17]
```diff
@@ -2,8 +2,306 @@
...
+| `helium_chartboost_sdk` | Helium Chartboost |
```
Comment (uerceg) [inline src/content/docs/sdk/react-native/features/ad-revenue.mdx:43]:
```suggestion
| `"helium_chartboost_sdk"` | Helium Chartboost |
```

File: src/content/docs/sdk/react-native/features/ad-revenue.mdx [hunk 18]
```diff
@@ -2,8 +2,306 @@
...
+| `'adx_sdk'`             | Ad(X)             |
```
Comment (uerceg) [inline src/content/docs/sdk/react-native/features/ad-revenue.mdx:44]:
```suggestion
| `"adx_sdk"`             | Ad(X)             |
```

File: src/content/docs/sdk/react-native/features/ad-revenue.mdx [hunk 19]
```diff
@@ -2,8 +2,306 @@
...
+| `'publisher_sdk'`       | Generic source    |
```
Comment (uerceg) [inline src/content/docs/sdk/react-native/features/ad-revenue.mdx:45]:
```suggestion
| `"publisher_sdk"`       | Generic source    |
```

File: src/content/docs/sdk/react-native/features/ad-revenue.mdx [hunk 20]
```diff
@@ -2,8 +2,306 @@
...
+| `'tradplus_sdk'`        | TradPlus          |
```
Comment (uerceg) [inline src/content/docs/sdk/react-native/features/ad-revenue.mdx:46]:
```suggestion
| `"tradplus_sdk"`        | TradPlus          |
```

File: src/content/docs/sdk/react-native/features/ad-revenue.mdx [hunk 21]
```diff
@@ -2,8 +2,306 @@
...
+| `'topon_sdk'`           | TopOn             |
```
Comment (uerceg) [inline src/content/docs/sdk/react-native/features/ad-revenue.mdx:47]:
```suggestion
| `"topon_sdk"`           | TopOn             |
```

File: src/content/docs/sdk/react-native/features/att.mdx [hunk 22]
```diff
@@ -2,8 +2,163 @@
...
+```ts
+getAppTrackingAuthorizationStatus: (callback: (authorizationStatus: number) => void) => void
+```
+
+</CodeBlock>
+
+You can retrieve a user's current authorization status at any time. Call the `getAppTrackingAuthorizationStatus` method to return the authorization status code as a `number`.
+
+```jsx
+Adjust.getAppTrackingAuthorizationStatus();
```
Comment (uerceg) [inline src/content/docs/sdk/react-native/features/att.mdx:116]:
```suggestion
Adjust.getAppTrackingAuthorizationStatus(function (status) {
    // check status value
});
```

File: src/content/docs/sdk/react-native/features/att.mdx [hunk 23]
```diff
@@ -2,8 +2,163 @@
...
+```
+
+### [Example](example-1)
+
+This example shows how to collect the user's authorization status and convert it to a `string`. This information is assigned to a variable called `authorizationStatus` and passed as a session partner parameter with the key `"status"`.
+
+```jsx
+Adjust.getAppTrackingAuthorizationStatus(function(status) {
+      Adjust.addGlobalCallbackParameter("status", status)
```
Comment (uerceg) [inline src/content/docs/sdk/react-native/features/att.mdx:125]:
```suggestion
      Adjust.addGlobalCallbackParameter("status", status.toString());
```

File: src/content/docs/sdk/react-native/features/att.mdx [hunk 24]
```diff
@@ -2,8 +2,163 @@
...
+});
+```
+
+## [Custom prompt timing](custom-prompt-timing)
+
+<CodeBlock title="Method signature">
+
+```js
```
Comment (uerceg) [inline src/content/docs/sdk/react-native/features/att.mdx:133]:
```suggestion
```ts
```

File: src/content/docs/sdk/react-native/features/att.mdx [hunk 25]
```diff
@@ -2,8 +2,163 @@
...
+AdjustConfig.prototype.setAttConsentWaitingInterval = function(attConsentWaitingInterval);
```
Comment (uerceg) [inline src/content/docs/sdk/react-native/features/att.mdx:134]:
```suggestion
public setAttConsentWaitingInterval(attConsentWaitingInterval: number): void
```

File: src/content/docs/sdk/react-native/features/callbacks.mdx [hunk 26]
```diff
@@ -22,11 +25,10 @@ Session callbacks have access to a response data object. You can use its propert
...
 | Property       | Data type                | Description                                                            |
 | -------------- | ------------------------ | ---------------------------------------------------------------------- |
-| `Message`      | `string`                 | The message from the server or the error logged by the SDK.            |
-| `Timestamp`    | `string`                 | The timestamp from Adjust's servers.                                   |
-| `Adid`         | `string`                 | A unique device identifier provided by Adjust.                         |
-| `JsonResponse` | `object<string, object>` | The JSON object with the response from the server.                     |
-| `WillRetry`    | `boolean`                | Indicates whether there will be an attempt to resend a failed package. |
+| `message`      | `string`                 | The message from the server or the error logged by the SDK.            |
+| `timestamp`    | `string`                 | The timestamp from Adjust's servers.                                   |
+| `jsonResponse` | `object<string, object>` | The JSON object with the response from the server.                     |
```
Comment (uerceg) [inline src/content/docs/sdk/react-native/features/callbacks.mdx:30]:
```suggestion
| `jsonResponse` | `string` | The JSON string with the response from the server.                     |
```

File: src/content/docs/sdk/react-native/features/callbacks.mdx [hunk 27]
```diff
@@ -50,7 +52,266 @@ const adjustConfig = new AdjustConfig(
...
+
+<Table>
+
+| Property       | Data type                | Description                                                            |
+| -------------- | ------------------------ | ---------------------------------------------------------------------- |
+| `message`      | `string`                 | The message from the server or the error logged by the SDK.            |
+| `timestamp`    | `string`                 | The timestamp from Adjust's servers.                                   |
+| `eventToken`   | `string`                 | The event token                                                        |
+| `callbackId`   | `string`                 | The custom callback ID set on the event object                         |
+| `jsonResponse` | `object<string, object>` | The JSON object with the response from the server.                     |
```
Comment (uerceg) [inline src/content/docs/sdk/react-native/features/callbacks.mdx:156]:
```suggestion
| `jsonResponse` | `string` | The JSON string with the response from the server.                     |
```

File: src/content/docs/sdk/react-native/features/privacy.mdx [hunk 28]
```diff
@@ -134,7 +137,415 @@ adjustThirdPartySharing.addGranularOption(
...
+
+-  `urlStrategyDomains` (`any`): The country or countries of data residence, or the endpoints to which you want to send SDK traffic.
+-  `useSubdomains` (`boolean`): Whether the domain should be treated as an Adjust domain. If `true`, the SDK will prefix the domains with Adjust-specific subdomains. If `false`, the SDK will use the provided domain as-is, without adding any prefixes.
+-  `isDataResidency` (`boolean`): Whether the domain should be used for data residency.
+
+<Table>
+
+| IR: strategy              | Main and fallback domain          | Use sub domains | Is Data Residency |
+| ------------------------- | --------------------------------- | --------------- | ----------------- |
+| EU data residency         | `'eu.adjust.com'`                 | `true`          | `true`            |
```
Comment (uerceg) [inline src/content/docs/sdk/react-native/features/privacy.mdx:319]:
```suggestion
| EU data residency         | `"eu.adjust.com"`                 | `true`          | `true`            |
```

File: src/content/docs/sdk/react-native/features/privacy.mdx [hunk 29]
```diff
@@ -134,7 +137,415 @@ adjustThirdPartySharing.addGranularOption(
...
+| Turkish data residency    | `'tr.adjust.com'`                 | `true`          | `true`            |
```
Comment (uerceg) [inline src/content/docs/sdk/react-native/features/privacy.mdx:320]:
```suggestion
| Turkish data residency    | `"tr.adjust.com"`                 | `true`          | `true`            |
```

File: src/content/docs/sdk/react-native/features/privacy.mdx [hunk 30]
```diff
@@ -134,7 +137,415 @@ adjustThirdPartySharing.addGranularOption(
...
+| US data residency         | `'us.adjust.com'`                 | `true`          | `true`            |
```
Comment (uerceg) [inline src/content/docs/sdk/react-native/features/privacy.mdx:321]:
```suggestion
| US data residency         | `"us.adjust.com"`                 | `true`          | `true`            |
```

File: src/content/docs/sdk/react-native/features/privacy.mdx [hunk 31]
```diff
@@ -134,7 +137,415 @@ adjustThirdPartySharing.addGranularOption(
...
+| China global URL strategy | `'adjust.world'`, `'adjust.com'`  | `true`          | `false`           |
```
Comment (uerceg) [inline src/content/docs/sdk/react-native/features/privacy.mdx:322]:
```suggestion
| China global URL strategy | `"adjust.world"`, `"adjust.com"`  | `true`          | `false`           |
```

File: src/content/docs/sdk/react-native/features/privacy.mdx [hunk 32]
```diff
@@ -134,7 +137,415 @@ adjustThirdPartySharing.addGranularOption(
...
+| China URL strategy        | `'adjust.cn'`, `'adjust.com'`     | `true`          | `false`           |
```
Comment (uerceg) [inline src/content/docs/sdk/react-native/features/privacy.mdx:323]:
```suggestion
| China URL strategy        | `"adjust.cn"`, `"adjust.com"`     | `true`          | `false`           |
```

File: src/content/docs/sdk/react-native/features/privacy.mdx [hunk 33]
```diff
@@ -134,7 +137,415 @@ adjustThirdPartySharing.addGranularOption(
...
+| China only URL strategy   | `'adjust.cn'`                     | `true`          | `false`           |
```
Comment (uerceg) [inline src/content/docs/sdk/react-native/features/privacy.mdx:324]:
```suggestion
| China only URL strategy   | `"adjust.cn"`                     | `true`          | `false`           |
```

File: src/content/docs/sdk/react-native/features/privacy.mdx [hunk 34]
```diff
@@ -134,7 +137,415 @@ adjustThirdPartySharing.addGranularOption(
...
+| India URL strategy        | `'adjust.net.in'`, `'adjust.com'` | `true`          | `false`           |
```
Comment (uerceg) [inline src/content/docs/sdk/react-native/features/privacy.mdx:325]:
```suggestion
| India URL strategy        | `"adjust.net.in"`, `"adjust.com"` | `true`          | `false`           |
```

File: src/content/docs/sdk/react-native/features/skad.mdx [hunk 35]
```diff
@@ -2,8 +2,225 @@
...
+### [Example](example-1)
+
+This example shows how to log the following when the conversion value updates:
+
+-  A message confirming the conversion value update
+-  The new conversion value
+
+```jsx
+import { Adjust, AdjustEvent, AdjustConfig } from "react-native-adjust";
+import { AdjustOaid } from "react-native-adjust-oaid";
```
Comment (uerceg) [inline src/content/docs/sdk/react-native/features/skad.mdx:120]:
```suggestion
```

File: src/content/docs/sdk/react-native/features/skad.mdx [hunk 36]
```diff
@@ -2,8 +2,225 @@
...
+
+</Table>
+
+### [Example](example-3)
+
+This example shows how to log the the fine conversion value, the coarse conversion value, and whether the SKAdNetwork postback is set to send before the conversion window ends.
+
+```jsx
+import { Adjust, AdjustEvent, AdjustConfig } from "react-native-adjust";
+import { AdjustOaid } from "react-native-adjust-oaid";
```
Comment (uerceg) [inline src/content/docs/sdk/react-native/features/skad.mdx:169]:
```suggestion
```

**Comment by uerceg** [comment]
Code snippet for OAID plugin is calling `Adjust.create` instead of `Adjust.initSdk`.

File: src/content/docs/sdk/react-native/index.mdx [hunk 37]
```diff
@@ -21,19 +24,318 @@ To use the Adjust SDK in your React Native app, you need to add it to your proje
...
+
+<Callout type="important">
+
+When running tests, ensure that your environment is set to `AdjustEnvironment.sandbox`. Change this to `AdjustEnvironment.production` before you submit your application to the app store.
+
+</Callout>
+
+## [4. Set up Android devices](4-set-up-android-devices)
+
+### [Add permissions](add-permissions)
```
[the remark this answers is not in the document]
Comment (ivosam3) [reply src/content/docs/sdk/react-native/index.mdx:77]: Updated with your descriptions

**Pull request merged**